tools/test/audio/src_test.m

```diff
@@ -413,7 +413,9 @@
 %% Analyze
 test.fs = t.fs2;
 test = chirp_test_analyze(test);
-src_test_result_print(t, 'Chirp', 'chirpf');
+if ~test.fail
```

Comment:
Mistake here, the fail is -1 if the chirp file doesn't exist. Need to plot & store a failed chirp spectra.

Reply:
fixed now